src/TLuaInterpreter.cpp

```diff
@@ -2462,9 +2472,15 @@ int TLuaInterpreter::openUserWindow( lua_State *L )
     {
         luaSendText = lua_tostring( L, 1 );
     }
+
+    bool loadLayout = true;
+    if( ! lua_isnoneornil(L, 2) && lua_isboolean(L, 2) ) {
+        loadLayout = lua_toboolean(L, 2);
+    }
+
     Host * pHost = TLuaInterpreter::luaInterpreterMap[L];
     QString text(luaSendText.c_str());
```

Comment:
:warning: :earth_africa: __I18n__ if we do not yet, we will soon have potential non-ASCII Window/Console/Label names - and this constructor will fail once `QT_NO_CAST_FROM_ASCII` and `QT_NO_CAST_TO_ASCII` are `#define`d .

Not _necessarily_ an issue for this PR but something someone ought to note...